test_youtube_cookies.py

```
@@ -0,0 +1,96 @@
+    if not os.path.exists("cookies.txt"):
+        print("❌ Error: 'cookies.txt' not found in current directory.")
+        print("   Please place the 'cookies.txt' file in the same folder as this script.")
+        return
+
+    print("✅ 'cookies.txt' found.")
+
+    # 2. Initialize Processor & Summarizer
+    try:
+        processor = YouTubeProcessor()
+        summarizer = GeminiSummarizer()
+    except Exception as e:
```

Ruff (0.15.1)

[warning] Do not catch blind exception: `Exception`

(BLE001)